agent/util-scripts/pbench-tool-meister-start

```diff
@@ -290,7 +878,37 @@ def main(_prog, cli_params):
 
     sysinfo, bad_l = cli_verify_sysinfo(cli_params.sysinfo)
     if bad_l:
-        logger.error('invalid sysinfo option(s), "{}"', ",".join(bad_l))
+        logger.error("invalid sysinfo option(s), '{}'", ",".join(bad_l))
+        return ReturnCode.BADSYSINFO
+
+    if (cli_params.redis_server is not None and cli_params.tool_data_sink is None) or (
+        cli_params.redis_server is None and cli_params.tool_data_sink is not None
+    ):
+        logger.error(
+            "both --redis-server and --tool-data-sink must be specified if either is used"
+        )
+        return ReturnCode.MISSINGPARAMS
+
+    try:
+        redis_server = RedisServer(cli_params.redis_server)
+    except RedisServer.Err as exc:
+        logger.error("{exc}")
```

Comment:
Does this line do what you want it to do?

I would have thought that you would want one of these, instead:
```
        logger.error(f"{exc}")
```
or
```
        logger.error(str(exc))
```
or maybe
```
        logger.error(exc)
```
or something similar hitting an method or attribute on `exc`.

Ditto line 901.

Reply:
No, that is a bug, nice catch.